### PR TITLE
Add support for additional script tags

### PR DESCRIPTION
**Pull request by viglucci**
Adds support for a 5th parameter to the PrettyPageHandler constructor so that additional scripts can be included in the page when the ejs template is rendered. Also updated some use of quotes within modified files for consistency.

Addresses #10

### CODE REVIEW

File: handler/PrettyPageHandler.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ var util = require('util');
  * @param sendResponse
  * @class
```
Comment (harishanchu) [inline handler/PrettyPageHandler.js]: Can you include additionalScripts in jsdoc as an optional parameter.

File: package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "@positron/stack-trace": "1.0.0"
   },
   "devDependencies": {
+    "cheerio": "^1.0.0-rc.2",
```
Comment (harishanchu) [inline package.json]: RC versions of library as dev dependency is okay right?

Comment (viglucci) [reply package.json]: Should be fine, all depends on how the author of that library handles their versioning. As you mentioned though this is a dev dependency only needed for testing, so very minimal risk if any.

File: handler/PrettyPageHandler.js (hunk 1 above)
Comment (viglucci) [reply handler/PrettyPageHandler.js]: Addressed in latest commit. There seem to be a few ways to specify any optional parameter, so if there is another preferred style please let me know.

**Comment by viglucci** [comment]
Doesn't look like CI is triggering properly for the latest commit, as it still shows as pending.

**Comment by viglucci** [comment]
@harishanchu thanks for merging this!

When can I anticipate the next version to be published?

**Pull request merged**